app/verify/verify-content.js

```diff
@@ -0,0 +1,22 @@
+const crypto = require('crypto')
+
+const verifyContent = (content, hash) => {
+  try {
+    const fileHash = crypto.createHash('sha256').update(content).digest('hex')
```

Comment:
rule of 3, break into own function: https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)

Reply:
Agreed.  As is this function is doing two things, wise to break it down.